homeassistant/components/go2rtc/__init__.py

```diff
@@ -68,13 +69,23 @@
 
 
 CONFIG_SCHEMA = vol.Schema(
-    {DOMAIN: vol.Schema({vol.Optional(CONF_URL): cv.url})},
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Optional(CONF_URL): cv.url,
+                vol.Optional(CONF_USE_BUILTIN, default=True): cv.boolean,
```

Comment:
It's a bit weird to configure an integration to not do anything, but I think it's a better option than automatically unregistering the builtin provider if we detect a legacy provider, since we want to promote the builtin provider over legacy providers.

Reply:
We don't want to add this option now. It can be added later if needed.